### PR TITLE
Governance/index (en + fr)

### PR DESCRIPTION
**Pull request by Christian-CROCHE**
I don’t why there is a commit on home page _index.en.html included in this PR. I will revert the changes shown here as there are already included in the PR #109.

### CODE REVIEW

File: content/_index.en.html [hunk 1]
```diff
@@ -20,7 +20,7 @@ <h2 class="mb-5">Open Source Railway Designer</h2>
 
 
 {{% blocks/lead color="primary" %}}
-An open source web application for railway infrastructure design, capacity analysis, timetabling, and more
+An open source web application for railway infrastructure design, capacity analysis, timetabling and more!
```
Comment (Christian-CROCHE) [inline content/_index.en.html:23]:
```suggestion
An open source web application for railway infrastructure design, capacity analysis, timetabling, and more
```

Comment (Christian-CROCHE) [reply content/_index.en.html:23]: Revert to original file as these changes are already included in the PR #109.

File: content/_index.en.html [hunk 2]
```diff
@@ -46,7 +46,7 @@ <h2 class="mb-5">Open Source Railway Designer</h2>
 {{% /blocks/feature %}}
 
 {{% blocks/feature icon="fas fa-university" title="Open governance" url="./about/governance/" %}}
-OSRD is publicly designed, and decisions taken collectively
+OSRD is publicly designed, decisions are taken collectively
```
Comment (Christian-CROCHE) [inline content/_index.en.html:49]:
```suggestion
OSRD is publicly designed, and decisions taken collectively
```

Comment (Christian-CROCHE) [reply content/_index.en.html:49]: Revert to original file as these changes are already included in the PR #109.

**Review by axrolld — APPROVED** [review]

**Review by flomonster — APPROVED** [review]

**Pull request merged**